doc/source/management/audit_log_plugin.rst

```diff
@@ -196,28 +407,68 @@ When this variable is set to ``ON`` log file will be closed and reopened. This c
 
 This variable can be used to specify the size of memory buffer used for logging, used when :variable:`audit_log_strategy` variable is set to ``ASYNCHRONOUS`` or ``PERFORMANCE`` values. This variable has effect only when :variable:`audit_log_handler` is set to ``FILE``.
 
+.. variable:: audit_log_exclude_accounts
+
+    :version 5.6.38-78.0: Implemented
+    :cli: Yes
+    :scope: Global
+    :dyn: Yes
+    :vartype: String
+
+This variable is used to specify the list of users for which :ref:`filtering_by_user` is applied. The value can be ``NULL`` or comma separated list of accounts in form ``user@host`` or ``'user'@'host'`` (if user or host contains comma).
```

Comment:
If this variable is set, then audit_log_exclude_commands must be unset, and vice versa. - here and below